live/liveview.go

```diff
@@ -824,6 +829,16 @@ func PushNav(ctx context.Context, typ LiveNavType, path string, params url.Value
 	return nil
 }
 
+// Redirect sends a ("dead") redirect event to the View
```

Comment:
I think Phoenix uses "non-live" for "dead" views. Either way this could probably be a little more explicit:

```suggestion
// Redirect sends an event to the View that triggers a full page load to url.
```

Also, I'm assuming we don't support the `external` argument?

Reply:
I didn't look at the phoenix method... :). [Looking at it now](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#redirect/2), I don't fully grok.  Would your provide either `to` or `external`?

Reply:
Using a `url.URL` supports both "external" and "local" so I kinda like this api better.  wdyt?